base/build.gradle.kts

```diff
@@ -56,3 +61,9 @@ protobuf {
 tasks {
     excludeGoogleProtoFromArtifacts()
 }
+
+detekt {
+    buildUponDefaultConfig = true
+    baseline = file("config/quality/detekt-baseline.xml")
```

Comment:
What happens if these files are not there?

I'm asking so that we can use script plugins (e.g. from `build-logic`).

Reply:
OK, let's now assume we have `buildSrc` (so that we don't introduce too many changes with `build-logic`).

Can we have these general things done there? Together with checking for the presence of configuration files, please.